pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1989,7 +1989,7 @@ bool GSQLBackend::getComment(Comment& comment)
   }
 }
 
-void GSQLBackend::feedComment(const Comment& comment)
+bool GSQLBackend::feedComment(const Comment& comment)
```

Comment:
Always returning `true` feels a bit odd. If you're handling errors by throwing an exception, that should be enough.